selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/BrowserLogsUtil.java

```diff
@@ -25,27 +27,115 @@
  * Read and store browser logs to the test logs. Log level and type are defined in {@link
```

Comment:
Javadocs doesn't take into account performance logs. And this class stores nothing.